install/install.go

```diff
@@ -614,16 +619,16 @@ func (k *K8sInstaller) listVersions() error {
 }
 
 func (k *K8sInstaller) preinstall(ctx context.Context) error {
-	// If --list-versions flag is specified, print available versions and return.
-	if k.params.ListVersions {
-		return k.listVersions()
-	}
 	if err := k.autodetectAndValidate(ctx); err != nil {
 		return err
 	}
 
 	switch k.flavor.Kind {
 	case k8s.KindGKE:
+		// TODO (ajs): Confirm that k.params.HelmOps.MergeValues(...) here produces the same values as our local
```

Comment:
Note that both of these codepaths end up getting used, because here in `preinstal` we call `HelmOps.MergeValues`, and then later in `getHelmValues` we call our own implementation of `MergeValues`. So this is currently the expected use of both of these and should work for the time being.